src/components/blocks/search-form/_search-form.scss

```diff
@@ -415,28 +471,43 @@
     }
 
     &--date {
+      @include base.typography-typeSetting(2);
+
       border: none;
-      color: base.sitecolors-siteColor("vam-black");
-      display: block;
-      margin-top: 6px;
-      padding: 6px 4px 6px 14px;
-      width: 100px;
+      color: base.sitecolors-siteColor("vam-grey-3");
+      padding: 8px 12px;
     }
   }
 
   &__filter-select {
-    @include base.typography-typeSetting(2, "regular");
+    @include base.typography-typeSetting(4, "regular");
 
     appearance: none;
-    background: transparent;
-    border: 1px solid base.sitecolors-siteColor("vam-white");
+    background-color:base.sitecolors-siteColor("vam-grey-6");
+    border: 1px solid base.sitecolors-siteColor("vam-grey-6");
     border-radius: 2px;
-    color: base.sitecolors-siteColor("vam-white");
-    padding: 10px 40px 10px 20px;
+    color: base.sitecolors-siteColor("vam-black");
+    padding: 8px 40px 8px 16px;
     width: 100%;
 
     @include mixins.breakpoints-bpMinXSmall {
       width: auto;
+
+      .b-search-form--etc & {
+        width: 100%;
+      }
+    }
+
+    @include mixins.breakpoints-bpMinSmall {
+      .b-search-form--etc & {
+        border-radius: 2px 0 0 2px;
+      }
+    }
+
+    @include mixins.breakpoints-bpMinMedium {
+      @include base.typography-typeSetting(5, "regular");
+
+      padding: 12px 48px 12px 20px;
```

Comment:
all these `__filter-select styles` are affecting where it's used elsewhere, i.e. the **Location** select on the `etc-search` variant. You could apply these styles to this select specifically by defining them for a `b-search-form__filter-select` nested under `b-search-form__advanced-search-wrapper` further down this file.